pkg/provider/defaults/default_metric_providers.go

```diff
@@ -0,0 +1,42 @@
+/*
+Copyright 2023 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package defaults provides a default implementation of metrics providers.
+package defaults
+
+import (
+	"sigs.k8s.io/custom-metrics-apiserver/pkg/provider"
+)
+
+type DefaultExternalMetricsProvider struct{}
+
+func (em DefaultExternalMetricsProvider) ListAllExternalMetrics() []provider.ExternalMetricInfo {
+	return []provider.ExternalMetricInfo{
+		{
+			Metric: "externalmetric",
```

Comment:
resource names are always in plural form in k8s

Reply:
nice catch! I didn't notice it